selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/ImportWizardFormTest.java

```diff
@@ -0,0 +1,614 @@
+      throws IOException, JsonParseException {
+    projectExplorer.waitProjectExplorer();
+    menu.runCommand(
+        TestMenuCommandsConstants.Workspace.WORKSPACE,
+        TestMenuCommandsConstants.Workspace.IMPORT_PROJECT);
+    importProject.waitMainForm();
+    loader.waitOnClosed();
+    importProject.selectGitSourceItem();
+    loader.waitOnClosed();
+    importProject.typeURi(url);
+    importProject.typeProjectName(projectName);
+    importProject.waitBranchIsNotSelected();
+    importProject.clickBranchCheckbox();
+    importProject.waitBranchIsSelected();
+    importProject.typeBranchName(branchName);
+    importProject.clickImportBtn();
+    importProject.waitMainFormIsClosed();
+    loader.waitOnClosed();
+    projectWizard.waitCreateProjectWizardForm();
+    projectWizard.selectTypeProject(MAVEN);
+    loader.waitOnClosed();
+    projectWizard.clickSaveButton();
+    loader.waitOnClosed();
+    projectWizard.waitCreateProjectWizardFormIsClosed();
+    projectExplorer.waitProjectExplorer();
+  }
+
+  private void expandDirectoryMyLib(String projectName) throws Exception {
+    projectExplorer.expandPathInProjectExplorerAndOpenFile(
+        projectName + "/my-lib/src/main/java/hello", "SayHello.java");
+    editor.waitActive();
+    projectExplorer.expandPathInProjectExplorerAndOpenFile(
+        projectName + "/my-lib/src/test/java/hello", "SayHelloTest.java");
+    editor.waitActive();
+  }
+
+  private void initRepoForFirstTest() throws IOException {
+    keepDirectoryRepo.addContent(
+        Paths.get(getClass().getResource("/projects/java-multimodule").getPath()));
+  }
+
+  private void initRepoForSecondTest() throws IOException {
+    importBranchRepo.addContent(
+        Paths.get(getClass().getResource("/projects/default-spring-project").getPath()));
+
+    importBranchRepo.createBranch(FIRST_BRANCH);
+    importBranchRepo.createBranch(SECOND_BRANCH);
+    importBranchRepo.createBranch(THIRD_BRANCH);
+
+    importBranchRepo.addContent(
+        Paths.get(getClass().getResource("/projects/Repo_For_Test_branch1").getPath()),
+        THIRD_BRANCH);
+  }
+
+  private void initRepoForThirdTest() throws Exception {
+    submoduleRepo.addContent(
+        Paths.get(getClass().getResource("/projects/java-multimodule").getPath()));
+    submoduleRepo.addSubmodule(
+        Paths.get(getClass().getResource("/projects/Repo_For_Test").getPath()),
+        FIRST_SUBMODULE_NAME);
+    submoduleRepo.addSubmodule(
+        Paths.get(getClass().getResource("/projects/testRepo").getPath()), SECOND_SUBMODULE_NAME);
+  }
+
+  private void loginToGitHub() {
+    try {
+      askDialog.waitFormToOpen(25);
+    } catch (TimeoutException ex) {
+      importProject.closeWithIcon();
+      events.clickEventLogBtn();
+      events.waitExpectedMessage("Failed to authorize application on GitHub");
+      fail("Known issue https://github.com/eclipse/che/issues/8288", ex);
```

Comment:
It seems outdated verification because https://github.com/eclipse/che/issues/8288 had been closed.